scastiel/my-movie-library
# Add search feature in app header (ABCD-1234)

_[@Jim](#), I added you as a reviewer because I think you can help us, but don’t feel pressured if you’re too busy._

Jira issue: [ABCD-1234](#)
Figma design: [link](#)

Add a new section at the right of the app header when the user is signed in, containing a search form (input + button). When submitted, the form redirects to the search page.

## Screenshots

![Screenshot 2023-10-16 at 14 00 39](https://github.com/scastiel/my-movie-library/assets/301948/18c56f7b-0342-4d0b-8fd9-2ae7a629c2e2)

## Steps to test:

1. Sign in with a test user
2. Enter a query in the search bar at the top right, e.g. “test”
3. Press enter or the search button
4. You should be redirected to the search page (still empty for now), with the query in the URL (`query` parameter)

## Next steps:

- Implement the search page ([ABCD-1235](#))
- Update end-to-end tests ([ABCD-1236](#)) in repository e2e-tests
